### ISSUE TITLE
x/gamm: use `Example` of cobra.Command for `create-pool` cli

### ISSUE THREAD

**Issue by mattverse** [osmosis-labs/osmosis#1325]
The `create-pool` cli used in gamm module requires a json file to create a pool. 

We recently figured out that we could use the `Example` command in cobra.Command (implemented in clawback vesting account cli). We could also do the same thing for `create-pool` to provide better examples for the required json to create a pool instead of relying on `Long`.

**Comment by alexanderbez** [comment]
Yes! More usage of `Example` rather than silly `Sprintf` in `Long 🙌
